src/pac_exttype.h

```diff
@@ -21,6 +21,7 @@ class ExternType : public Type {
     string EvalMember(const ID* member_id) const override;
     bool IsNumericType() const override { return ext_type_ == NUMBER; }
     bool IsPointerType() const override { return ext_type_ == POINTER; }
+    bool IsBooleanType() const override { return ext_type_ == BOOLEAN; }
```

Comment:
If `IsNumericType()` was previously true for Booleans and now is not, I wonder if that could matter. I searched a bit and the only place that caught my eye was [this bit](https://github.com/zeek/binpac/blob/master/src/pac_type.h#L113) in pac_type.h, which I think needs an exclusion also for `IsBooleanType()`.